forge/routes/api/project.js

```diff
@@ -170,15 +170,16 @@ module.exports = async function (app) {
             return
         }
 
-        const name = request.body.name
-
-        if (bannedNameList.includes(name)) {
+        const name = request.body.name?.trim()
+        const safeName = name?.toLowerCase()
+        if (bannedNameList.includes(safeName)) {
             reply.status(409).type('application/json').send({ error: 'name not allowed' })
-            return
+            return reply
```

Comment:
Why are we now returning `reply`?

Reply:
Something I read - 

> Since v4, we are making it mandatory to `return reply`. if you want to call `reply.send` after the route handler has completed its execution.

So thought it no harm to think ahead.  However, I will remove them & we can handle that at a future point (if ever)

Reply:
resolved in 5f0249b

Reply:
OK, I've found the comment in the fastify migration guide, but it only applies to async responses which we don't have.